components/header.js

```
@@ -0,0 +1,114 @@
+      }}
+    >
+      {includeMeta && (
+        <Meta as={Head} title={title} description={desc} image={img} />
+      )}
+      <Container variant="copy">
+        <Heading
+          as="h1"
+          variant="title"
+          sx={{ color: 'primary', '@media print': { fontSize: 5 } }}
+          children={title}
+        />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)